cart/style.css

```diff
@@ -3,13 +3,21 @@ body {
     flex-direction: column;
     justify-content: center;
     align-items: center;
+    background-image: url('/assets/background6.jpeg');
```

Comment:
this isn't loading in your site....I'm guessing pathing. Github is Linux based so it might want ./ or ../
I'm not sure until you reupload it to github.